### DISCUSSION TITLE
Add useful information about "Live" connections to the Data Source Sidebar

### DISCUSSION THREAD

**Discussion by 2metres** [foxglove/community#322] [Q&A]
Currently the Data Source Sidebar shows start, end, and duration for local files. Could we add some concept of "connection quality" or a sparkline with the messages coming through (a la a heartbeat/pulse)?

**Comment by 2metres** [comment]
@esthersweon 

Still think we could save space, not duplicate text, and make copy more readable by having one "Time" heading.

e.g. 
```
TIME
2018-07-23
8:28:47.647 - 8:29:06.797 PM PDT (19.1 sec)
```
instead of the 3 headings below:

<img width="320" alt="Screen Shot 2022-03-30 at 2 39 37 PM" src="https://user-images.githubusercontent.com/6993359/160935527-f4542be6-f513-4e54-a933-74e6a89cba23.png">

For instances where it spans AM / PM, we could do:
```
TIME
2018-07-23
8:28:47.647 AM - 8:29:06.797 PM PDT (12 hrs, 19.1 sec)
```

And for instances where it spans more than 1 day (not common), we could do:

```
TIME
2018-07-23 8:28:47.647 PM PDT - 
2018-07-24 8:29:06.797 PM PDT (1 day, 19.1 sec)
```

**Reply by esthersweon** [reply]
@2metres - I like your proposed formatting! The only change I'd make is to keep duration on its own line, so it's easy to scan for.
